runtime/tests/clinkerffi/downcall.h

```diff
@@ -27,457 +27,381 @@
  * Created by jincheng@ca.ibm.com
  */
 
+#if !defined(downcall_h)
+
```

Comment:
Of course, this needs `#define downcall_h` for the `#if` to be effective.

Reply:
Follow the existing coding convention for header to replace with:
```
#ifndef DOWNCALL_H
#define DOWNCALL_H

#endif /* DOWNCALL_H */
```